src/pdf_beamtime/src/inner_state_machine.cpp

```diff
@@ -0,0 +1,132 @@
+/*Copyright 2023 Brookhaven National Laboratory
+BSD 3 Clause License. See LICENSE.txt for details.*/
+#include <pdf_beamtime/inner_state_machine.hpp>
+
+InnerStateMachine::InnerStateMachine(
+  const rclcpp::Node::SharedPtr node, State external_state_enum)
+: node_(node), external_state_enum_(external_state_enum)
+{
+  internal_state_enum_ = Internal_State::RESTING;
+}
+
+InnerStateMachine::~InnerStateMachine()
+{
+}
```

Comment:
Is this destructor necessary here?

Reply:
It's not. At first, I kept it thinking I might need it down the line. It is removed.
https://github.com/maffettone/erobs/blob/6b416cd1616354090b0c778013ed71d6864d5fa5/src/pdf_beamtime/src/inner_state_machine.cpp#L5-L12